docs/testplan/platform/pmon-test-plan.md

```diff
@@ -0,0 +1,227 @@
+   ```
+
+## Test case # 3 Data for lm-sensors daemon
+### Test objective
+Verify the expected data for lm-sensors daemon
+   - condition to start : "skip_sensors" != true and [ -e "/usr/share/sonic/platform/sensors.conf" ]
+### Test steps
+1. Verify the lm-sensors running status
+   - If it's not running, report the error status and go to the next 
+
+## Test case # 4 Data for fancontrol daemon
+### Test objective
+Verify the expected data for fancontrol daemon
+   - condition to start : "skip_fancontrol" != true and [ -e "/usr/share/sonic/platform/fancontrol" ]
+### Test steps
+1. Verify the fancontrol running status
+   - If it's not running, report the error status and go to the next 
+
+## Test case # 5 Data for ledd daemon
+### Test objective
+Verify the expected data for ledd daemon
+   - condition to start : "skip_ledd" != true
+### Test steps
+1. Verify the ledd running status
+   - If it's not running, report the error status and go to the next 
+
+## Test case # 6 Data for pcied daemon
+### Test objective
+Verify the expected data for pcied daemon
+   - condition to start : "skip_pcied" != true
+### Test steps
+1. Verify the pcied running status
+   - If it's not running, report the error status and go to the next 
+2. Verify the pcie config file "/usr/share/sonic/platform/pcie.yaml" exists
+3. Verify the "PCIE_DEVICES" has "status" field set to "SUCCESS" in state DB
+
+## Test case # 7 Data for psud daemon
+### Test objective
+Verify the expected data for psud daemon
+   - condition to start : "skip_psud" != true
+### Test steps
+1. Verify the pcied running status
+   - If it's not running, report the error status and go to the next 
+2. Verify the 'PSU_INFO|PSU {}' is/are available in state DB
+3. Verify if the data for each 'PSU {}' is expected to compare "/usr/share/sonic/platform/platform.json"
+4. Verify if the status of each PSU is "OK"
+
+## Test case # 8 Data for syseepromd daemon
+### Test objective
+Verify the expected data for syseepromd daemon
+   - condition to start : "skip_syseepromd" != true
+### Test steps
+1. Verify the pcied running status
+   - If it's not running, report the error status and go to the next 
+2. Verify the 'EEPROM_INFO|State' in state DB has the 'Initialized' field set to '1'
+3. Verify the 'EEPROM_INFO|*' has valid pairs of key and value in state DB
+   - Verify the data to check with the output of the platform api or cli command
+
+## Test case # 9 Data for thermalctld daemon
+### Test objective
+Verify the expected data for thermalctld daemon
+   - condition to start : "skip_thermalctld" != true
+### Test steps
+1. Verify the pcied running status
+   - If it's not running, report the error status and go to the next 
+2. Verify the policy file "/usr/share/sonic/platform/thermal_policy.json" exists
+3. Verify the 'FAN_INFO|{}' and 'TEMPERATURE_INFO|{}' available in state DB
+   - Verify the data to check with the output of the platform api or cli command
+
+## Test case # 10 Data for xcvrd daemon
```

Comment:
please check whether current  **test_xcvr_info_in_db.py** can satisfy this test https://github.com/Azure/sonic-mgmt/blob/master/tests/platform_tests/test_xcvr_info_in_db.py